hmpps-sqs-spring-boot-autoconfigure/src/test/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueHealthTest.kt

```diff
@@ -58,6 +58,7 @@ class HmppsQueueHealthTest {
   @Test
   fun `should show status DOWN`() {
     whenever(sqsClient.getQueueUrl(anyString())).thenThrow(QueueDoesNotExistException::class.java)
+    whenever(sqsDlqClient.getQueueUrl(dlqName)).thenThrow(NullPointerException::class.java)
```

Comment:
I don't understand what this is for - the test is for the queue being down which doesn't concern the dlq.  Was there some other reason for putting this in?  (Same for rest of this class).

Reply:
They should be RuntimeException. They initially weren't needed as the NullpointerException that was initially thrown (as no dlqclient) worked perfectly.
However, because we now allow the dlqclient to be null - the exception wasn't being thrown (it was happy just to return null)